### PR TITLE
3.0 #5 Added key for isAuthenticationRequest to server container profile.

### PR DESCRIPTION
**Pull request by arjantijms**
#5 Added key for isAuthenticationRequest to server container profile.

Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: spec/src/main/asciidoc/revision-history.adoc [hunk 1]
```diff
@@ -220,3 +220,10 @@ the request
...
 . In `AuthConfig`, and `AuthConfigProvider` interfaces, removed incorrect assertion from javadoc of refresh method that checked `AuthException` could be thrown, and changed javadoc to indicate that the conditions for which `refresh` was expected to throw an `AuthException` should instead be handled within its existing declaration of throwing an (unchecked) `SecurityException`.
+
+=== Changes in Jakarta Authentication 3.0
+
+==== Changes to Servlet Container Profile
+
+ . Added the `jakarta.servlet.http.isAuthenticationRequest` key, so modules can distinguish between being called at the very start of a request and in the middle of it following a call to `HttpServletRequest.authenticate`. This could be needed to determine if certain contexts are certainly not active (such as the Faces context in Jakarta Faces).
```
Comment (ggam) [inline spec/src/main/asciidoc/revision-history.adoc:228]: > This could be needed to determine if certain contexts are certainly not active

Perhaps it's me, but that phrase, "verifying that a context is inactive" looks a bit cumbersome. I'd rephrase it to the positive version:

> This could be needed to determine if certain contexts are effectively active

Comment (arjantijms) [reply spec/src/main/asciidoc/revision-history.adoc:228]: Sure, thx!
